README.md

```diff
@@ -168,6 +171,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Three Monks](http://www.threemonk.com/) | New Baneshwor, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
 - [Truemark](https://www.truemark.com.np/) | Bagbazaar, Kathmandu | Software Development Company
 - [UBA Solutions](http://www.uba-solutions.com/) | Thadodhunga, Lalitpur | Software Research and Development
+- [Umanga Creation Pvt. Ltd.](https://umangacreation.com.np/) | Lazimpat-2 Uttar Dhoka, Kathmandu |We innovate/create/implement brands to realise their true potential
```

Comment:
@pradip7374 Thank you for the pull request. I am requesting you a few changes.

Only `Lazimpat, Kathmandu` or `Uttar Dhoka, Kathmandu`. Let's remain consistent with other companies on the list.

Need a space after `|`. Remove `We`.

```diff
- We
```

Can you make the description like this? `Innovate and create brands to realize their true potential`